src/Labels/Labels.php

```diff
@@ -279,6 +286,22 @@ private function getGoodbitsLabels(): array
 		];
 	}
 
+	/**
+	 * Return labels - Active Campaign
+	 *
+	 * @return array<string, string>
+	 */
+	private function getActiveCampaignLabels(): array
+	{
+		return [
+			'activeCampaignInvalidEmailError' => \__('Enter a valid email address.', 'eightshift-forms'),
+			'activeCampaignDuplicateError' => \__('Email address already exists in the system.', 'eightshift-forms'),
+			'activeCampaign500Error' => \__('There was and internal server error. Please try again.', 'eightshift-forms'),
+			'activeCampaignForbidenError' => \__('It looks like this form is not authorized to make this request please check your API key and try again.', 'eightshift-forms'),
```

Comment:
```suggestion
			'activeCampaignForbidenError' => \__('It looks like this form is not authorized to make this request. Please check your API key and try again.', 'eightshift-forms'),
```

Reply:
Authorization is API key-level, not form-level?

```suggestion
			'activeCampaignForbidenError' => \__('It looks like this API key is not authorized to make this request. Please check your API key and try again.', 'eightshift-forms'),
```